### PR TITLE
Update pt_br.json

### PR DESCRIPTION
**Pull request by ghost**
Improved translations.

### CODE REVIEW

File: src/main/resources/assets/jei/lang/pt_br.json [hunk 1]
```diff
@@ -2,60 +2,60 @@
   "_comment": "Tooltips",
   "jei.tooltip.config": "Configurações do JEI",
   "jei.tooltip.show.recipes": "Mostrar receitas",
-  "jei.tooltip.show.all.recipes": "Mostrar todas as receitas",
+  "jei.tooltip.show.all.recipes": "Todas as receitas",
   "jei.tooltip.delete.item": "Clique para excluir",
-  "jei.tooltip.liquid.amount.with.capacity": "%s / %s mB",
-  "jei.tooltip.liquid.amount": "%s mB",
+  "jei.tooltip.liquid.amount.with.capacity": "%,d / %,d mB",
+  "jei.tooltip.liquid.amount": "%,d mB",
```
Comment (Greg-21) [inline src/main/resources/assets/jei/lang/pt_br.json]: Is there a reason why you have backed out changes from commit fc2e33694fa9be82c488e794f278aba0e2874031? I suppose you did it accidentally.

Comment (ghost) [reply src/main/resources/assets/jei/lang/pt_br.json]: yeah, thanks for pointing that out

Comment (mezz) [reply src/main/resources/assets/jei/lang/pt_br.json]: @DaDodger this has not been resolved yet, maybe you didn't push your commit?

Comment (ghost) [reply src/main/resources/assets/jei/lang/pt_br.json]: @mezz I guess it's all good now. Ready to merge.

**Pull request closed without merging**